modules/kubernetes/linkerd/charts/linkerd-extras/templates/wait-for-certs.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: v1
```

Comment:
Is this still needed when the secret is created with Terraform?

Reply:
I want to make sure all certificates are issued before we install the helm chart for linkerd, making sure that we don't start the control plane before they are issued.

Sounds OK?

Reply:
Removed it according to call.